Packages/HBMihoyoAPI/Sources/HBMihoyoAPI/Common/Models/Server.swift

```diff
@@ -60,7 +60,19 @@ extension Server: Codable {}
 
 extension Server: CustomStringConvertible {
     public var description: String {
```

Comment:
i18n here? Or just leave it?

Reply:
i18n is not needed here. same as that in ophelper